### DISCUSSION TITLE
这是星核猎手第一杀手，你敢和她对视十秒吗

### DISCUSSION THREAD

**Discussion by Griffin233** [share121/inter-knot#721] [其他]
![StarRail_Image_1723275338](https://github.com/user-attachments/assets/e671acba-b087-43e5-8ea5-ec456a81b0c9)
这是星核猎手第一杀手萨缪尔，你敢和她对视十秒吗
如果超过五秒，说明你的冷血指数很高，超过十秒说明你的冷血指数爆表

**Comment by yang1cong2** [comment]
😐

**Comment by Akashic007** [comment]
太可怕了
![119425498_p0_master1200](https://github.com/user-attachments/assets/5a966386-6516-4acd-8230-efc0c54d1d15)

**Comment by AusertDream** [comment]
霍霍可爱捏↑  :)
